ament_cmake_cppcheck/cmake/ament_cppcheck.cmake

```diff
@@ -38,6 +40,9 @@ function(ament_cppcheck)
   set(result_file "${AMENT_TEST_RESULTS_DIR}/${PROJECT_NAME}/${ARG_TESTNAME}.xunit.xml")
   set(cmd "${ament_cppcheck_BIN}" "--xunit-file" "${result_file}")
   list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
+  if(ARG_INCLUDE_DIRS)
+    list(APPEND cmd "--include_dirs" "${ARG_INCLUDE_DIRS}")
```

Comment:
You might want to try this with multiple include dirs which contain spaces in the path.

Reply:
Appears to work.